prime-router/metadata/fhir_mapping/hl7/datatype/HDToLocation.yml

```diff
@@ -0,0 +1,72 @@
+resourceType: Location
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+name:
+  valueOf: HD.1
+  type: STRING
+  expressionType: HL7Spec
+
+identifier_1:
+  expressionType: nested
+  generateList: true
+  vars:
+    universalId: String, HD.3
+  expressionsMap:
+    value_1:
+      condition: $universalId EQUALS ISO || $universalId EQUALS UUID
```

Comment:
Is there significance to having the condition here vs at the top level? Want to be sure we don't add an empty identifier if the condition is false.

Reply:
Mostly just wanted to be consistent, though long term the value would be if there was another field that needed to be added to the identifier, that had nothing to to with the condition for `value`